Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -101,12 +78,19 @@ int RunFuzzTestingMain() {
   // The corpus location for the fuzzing target.
   NSString *corpus_location;
 
+  // Fuzzing target method, equivalent to LLVMFuzzerTestOneInput. This variable
+  // holds a pointer to the fuzzing method that is called by the fuzzing driver
+  // thousands of times with different inputs. Any method assigned to this
+  // variable must have the same signature as LLVMFuzzerTestOneInput: returns an
+  // int and takes two parameters: (const uint8_t *Data, size_t Size).
+  fuzzer::UserCallback llvm_fuzzer_test_one_input_method;
```

Comment:
Nit: this variable's name seems to be a little too detailed. Perhaps just `fuzzer_function`, `fuzzer_entry_point`, `fuzzer_callback` or similar?

Reply:
OK, `fuzzer_function` sounds good.